components/Circle/Circle.js

```diff
@@ -23,7 +23,7 @@ export default function Circle({ circleSize, name, color, count }) {
         overflow: "hidden",
         height: circleSize,
         borderRadius: "50%",
-        margin: "5px",
+        margin: "4px",
```

Comment:
I would say no inline styles but since it is allready in your main, i guess it has to be inline because of framer motion?

Reply:
motion.divs are kind of incompatible with styled components, thats why...